### ISSUE TITLE
Social Media post cover image gets cropped

### ISSUE THREAD

**Issue by alvarosaburido** [forem/forem#9550]
**Describe the bug**

Following the [Editor Guide](https://dev.to/p/editor_guide) the size for the correct image is:

> cover_image: cover image for post, accepts a URL.
> The best size is 1000 x 420.

Using an image of that size, for example

![this](https://dev-to-uploads.s3.amazonaws.com/i/0b3vsz5h4m25bn4fhmiv.png) 

If you share the post, social preview gets the `og:image` and twitter image cropped.

<img width="990" alt="Screenshot 2020-07-28 at 09 54 03" src="https://user-images.githubusercontent.com/4699008/88635707-477aa000-d0b8-11ea-93eb-4ac0e0e7a3de.png">

**To Reproduce**

1. Go  to this [post](https://dev.to/alvarosaburido/the-magic-of-svg-clip-path-1lf0) 
2. Open the link in [Facebook Sharing Debugger](https://developers.facebook.com/tools/debug)
3. Check `og:image` url.
Optional: Share post on facebook or twitter

If you check the `og:image` URL it has a series of cloudinary transformations, the last one forces the image  to  be have a height  of  500 `h_500`:

https://res.cloudinary.com/practicaldev/image/fetch/s--sMkrMxIz--/c_imagga_scale,f_auto,fl_progressive,h_500,q_auto,w_1000/https://res.cloudinary.com/practicaldev/image/fetch/s--X9GWXwyS--/c_imagga_scale%2Cf_auto%2Cfl_progressive%2Ch_420%2Cq_auto%2Cw_1000/https://dev-to-uploads.s3.amazonaws.com/i/0b3vsz5h4m25bn4fhmiv.png

![](https://res.cloudinary.com/practicaldev/image/fetch/s--sMkrMxIz--/c_imagga_scale,f_auto,fl_progressive,h_500,q_auto,w_1000/https://res.cloudinary.com/practicaldev/image/fetch/s--X9GWXwyS--/c_imagga_scale%2Cf_auto%2Cfl_progressive%2Ch_420%2Cq_auto%2Cw_1000/https://dev-to-uploads.s3.amazonaws.com/i/0b3vsz5h4m25bn4fhmiv.png)

If this value is set to `h_420` as the  [Editor Guide](https://dev.to/p/editor_guide) the image is shown correctly

**Expected behavior**

The resulting image from Cloudinary should have  1000x420 look like:

[Image](https://res.cloudinary.com/practicaldev/image/fetch/s--sMkrMxIz--/c_imagga_scale,f_auto,fl_progressive,h_420,q_auto,w_1000/https://res.cloudinary.com/practicaldev/image/fetch/s--X9GWXwyS--/c_imagga_scale%2Cf_auto%2Cfl_progressive%2Ch_420%2Cq_auto%2Cw_1000/https://dev-to-uploads.s3.amazonaws.com/i/0b3vsz5h4m25bn4fhmiv.png)

**Desktop (please complete the following information):**

- OS: Catalina `v10.15.5`
- Browser: Chrome 
- Version: Version 84.0.4147.89

**Additional context**

If  you go to  `app/view_objects/articles/social_image.rb` line 10, height is set to `500` by default.

**Comment by citizen428** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @forem/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/forem/forem/blob/master/CONTRIBUTING.md).

**Comment by michael-tharrington** [comment]
Just to point out that a DEV mod (@devarshishimpi) noted that this was happening to them the other day.

![Screen Shot 2023-10-27 at 9 41 05 AM](https://github.com/forem/forem/assets/16007075/4509dbf6-0a53-43d8-82bd-89a6f701bdf1)

**Comment by mikeydorje** [comment]
Closing as the codebase has since been updated to use standard Open Graph dimensions for new content. If still reproducible, please open a new issue with current reproduction steps.
